openstack/python/tar/bin/openstack-describe-instances

```diff
@@ -32,7 +32,8 @@ class OpenStackDescribeInstances(DescribeInstancesCommand, OpenStackCommand):
                  2: 'Terminated',
                  3: 'Pending',
                  4: 'Unknown',
-                 5: 'Stopped'}
+                 5: 'Stopped',
+                 7: 'Error'}
```

Comment:
Actually your initial link to `latest` points to their `master`, where they've changed the implementation for `NodeState` https://github.com/apache/libcloud/blob/trunk/libcloud/compute/types.py#L214
I'm leaving the hardcoded mapping for now.  It will be OK till v0.20.1.  Then we have to revisit the mapping and make it dynamic following their new approach.